src/scenes/Home/ISplashPage.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import Paper from '@material-ui/core/Paper';
+import ReactMarkdown from 'react-markdown';
+
+/**
+ * @author Reece Dunham
```

Comment:
Remove. We don't use such javadoc style. You have Git history for that reason.

Reply:
Sorry, just have an IntelliJ extension that does it for me automatically 😅